src/plugins/intel_cpu/src/nodes/eye.cpp

```diff
@@ -132,7 +132,9 @@ void Eye::executeSpecified() {
             size_t start = 0;
             size_t end = 0;
             splitter(elementsCount, nthr, ithr, start, end);
-            memset(dst + start, 0, (end - start) * sizeof(T));
+            for (size_t i = start; i < end; ++i) {
```

Comment:
what is the motivation here?

Reply:
Actually, I heed them, because bfloat impl has changed:
```
/runner/_work/openvino/openvino/src/src/plugins/intel_cpu/src/nodes/eye.cpp:135:19: error: ‘void* memset(void*, int, size_t)’ clearing an object of non-trivial type ‘class ov::intel_cpu::bfloat16_t’; use assignment or value-initialization instead [-Werror=class-memaccess]
[ 44%] Building CXX object src/frontends/onnx/frontend/CMakeFiles/openvino_onnx_frontend.dir/src/op/thresholded_relu.cpp.o
  135 |             memset(dst + start, 0, (end - start) * sizeof(T));

```